dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/translators/DefaultSymbolToDocumentableTranslator.kt

```diff
@@ -908,13 +908,30 @@ internal class DokkaSymbolVisitor(
         }
     }
 
+    private fun KaSession.getAccessorSymbolDocumentation(symbol: KaPropertyAccessorSymbol): DocumentationNode? {
+        val documentation = getDocumentation(symbol) ?: return null
+        return documentation.removePropertyTag()
+    }
+
     private fun KaSession.getDocumentation(symbol: KaSymbol) =
         if (symbol.origin == KaSymbolOrigin.SOURCE_MEMBER_GENERATED)
             // a primary (implicit default) constructor  can be generated, so we need KDoc from @constructor tag
             getGeneratedKDocDocumentationFrom(symbol) ?: if(symbol is KaConstructorSymbol) getKDocDocumentationFrom(symbol, logger) else null
         else
             getKDocDocumentationFrom(symbol, logger) ?: javadocParser?.let { getJavaDocDocumentationFrom(symbol, it) }
 
+    /**
+     * Unwrap the documentation for property accessors from the [Property] wrapper if its present.
+     * Otherwise, documentation will not be rendered
+     */
+    private fun DocumentationNode.removePropertyTag(): DocumentationNode =
+        DocumentationNode(children.map { it.removePropertyTag() })
+
+    private fun TagWrapper.removePropertyTag(): TagWrapper = when (this) {
+        is Property -> Description(root)
+        else -> this
+    }
+
```

Comment:
Just an observation, unrelated to this PR: I think we could have similar problems in other places (not even related to kotlin-as-java), because we have the ability to document a property both via `@property` (translated to `Property`) and via a direct comment on it (which will be translated to `Description` AFAIK). Though, maybe those are somehow handled on the render side :)

BTW, it appears that we also have similar logic in [K1 analysis](https://github.com/Kotlin/dokka/blob/b49856b08a8e56cf7d272cab6a6b16a47b71afb6/dokka-subprojects/analysis-kotlin-descriptors-compiler/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/descriptors/compiler/translator/DefaultDescriptorToDocumentableTranslator.kt#L826-L843).